tests/test_check_size.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+echo "Running cc-snapshot interface tests..."
```

Comment:
this looks like leftovers from copying the other test script, need to update for the name here